src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/questionWithResponses.tag

```diff
@@ -56,9 +58,8 @@
           </c:forEach>
         </ul>
       </c:if>
-
       <c:if test="${not isRecipientNameHidden}">
-        <div class="col-sm-12 form-inline mobile-align-left">
+        <div class="evalueeLabel-${questionWithResponses.question.qnIndx} col-sm-12 form-inline mobile-align-left">
```

Comment:
I named it evalueeLabel since it is a static label that says Evaluee/Recipient. Maybe you have a better name in mind ? 

Yes I was thinking about this. The instructions won't appear if recipient's name is hidden. Currently the only thing I can of is having an empty div element particularly dedicated for instructions above the evalueeLabel div to completely eliminate dependence on this one. I can then manipulate it using JS ?

Reply:
Ok so I think it might be beneficial to add an empty div for instructions due to 2 reasons. First, we will no longer depend on the recipient for manipulating it. Second the requirement of instructions is not just for const sum questions. This feature can be extended to include other question types as well eg. rubric. So I think it will be a better choice in the longer run. Let me know what you think. I don't want to make changes before your approval